src/views/Resources/Resources.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 import { FC } from "react";
 import { PageHeader } from "../../components/PageHeader/PageHeader";
-import { UnderConstruction } from "../../components/UnderConstruction/UnderConstruction";
+import { ResourcesCard } from "../../components/LessonCard/LessonCard";
```

Comment:
Initially, I was worried that I thought you had created a completely separate ResourcesCard! Until I saw that this is just being aliased here. But therefore on that note, I don't think it is a good practice to do that. If it's a LessonCard component, then it should still be named a LessonCard as far as the import.